src/components/listbox/listbox.jsx

```diff
@@ -17,18 +18,44 @@ export default function Listbox(props) {
     }
   });
 
+  const refIndexOf = target => props.refs.findIndex(ref => ref.current && ref.current.contains(target));
+  const optionsWasSelected = (toggledRefIndex, isSelected) => {
+    let newSelectedIndexes = selectedIndexes.slice(0, selectedIndexes.length);
+
+    if (isSelected) {
+      newSelectedIndexes.push(toggledRefIndex);
+    } else {
+      const indexOfToggledRef = newSelectedIndexes.indexOf(toggledRefIndex);
+      newSelectedIndexes = newSelectedIndexes.splice(indexOfToggledRef, 1);
```

Comment:
`splice`'s output is the array of removed items. How is this working? I think it would return the same array:

```
let newSelectedIndexes = [42];
newSelectedIndexes = newSelectedIndexes.splice(0, 1);
```

Then `newSelectedIndexes` is still `[42]`.